python/pdstools/utils/cdh_utils.py

```diff
@@ -474,7 +473,7 @@ def auc_to_gini(auc: float) -> float:
     return 2 * safe_range_auc(auc) - 1
 
 
-def _capitalize(fields: Union[str, Iterable[str]]) -> List[str]:
+def _capitalize(fields: Union[str, Iterable[str]], extra:Optional[List[str]]=[]) -> List[str]:
```

Comment:
Same as previous comment - initializing a list here means the list remains shared across every invocation of this function